Cargo.toml

```diff
@@ -67,11 +70,11 @@ service = ["dep:tower", "dep:tower-service"]
 http1 = ["hyper/http1"]
 http2 = ["hyper/http2"]
 
-tokio = ["dep:tokio", "dep:socket2"]
+tokio = ["dep:tokio_wasi", "dep:socket2"]
 
 # internal features used in CI
 __internal_happy_eyeballs_tests = []
 
 [[example]]
 name = "client"
-required-features = ["client-legacy", "http1", "tokio"]
+required-features = ["client-legacy", "http1", "tokio_wasi"]
```

Comment:
You cannot modify it here; it needs to remain as 'tokio.' It is the name of the feature, not the dependency name

Reply:
Right, too much search and replace 😆